tests/test_mutable_dict.py

```diff
@@ -36,6 +38,8 @@ def test_mutable_dict(session):
     session.add(u := User(name="foo", addresses={
         "home": {"street": "123 Main Street", "city": "New York"},
         "work": "456 Wall Street",
+        "updated_at": datetime.now(),
+        "uuid": uuid.uuid4()
     }))
```

Comment:
I think we should also update the test here https://github.com/upsmith-dev/SQLAlchemy-Nested-Mutable/blob/main/tests/test_mutable_pydantic_type.py#L32